test_outputs/method_parameter_count_changed.output.ron

```diff
@@ -1,92 +1,148 @@
-        "visibility_limit": String("public")
-    },
-    {
-        "current_parameter_count": List([Uint64(3)]),
-        "method_name": String("moved_trait_provided_method"),
-        "method_visibility": String("public"),
-        "name": String("StructWithMethods"),
-        "non_matching_span_begin_line": List([Uint64(55)]),
-        "non_matching_span_filename": List([String("src/test_cases/parameter_count_changed.rs")]),
-        "old_parameter_count": Uint64(2),
-        "path": List([
-            String("test_crates"),
-            String("test_cases"),
-            String("parameter_count_changed"),
-            String("StructWithMethods")
-        ]),
-        "span_begin_line": Uint64(41),
-        "span_filename": String("src/test_cases/parameter_count_changed.rs"),
-        "visibility_limit": String("public")
-    },
-    {
-        "current_parameter_count": List([Uint64(1)]),
-        "method_name": String("moved_method"),
-        "method_visibility": String("public"),
-        "name": String("StructWithMethods"),
-        "non_matching_span_begin_line": List([Uint64(66)]),
-        "non_matching_span_filename": List([String("src/test_cases/parameter_count_changed.rs")]),
-        "old_parameter_count": Uint64(2),
-        "path": List([
-            String("test_crates"),
-            String("test_cases"),
-            String("parameter_count_changed"),
-            String("StructWithMethods")
-        ]),
-        "span_begin_line": Uint64(47),
-        "span_filename": String("src/test_cases/parameter_count_changed.rs"),
-        "visibility_limit": String("public")
-    }
-]
+{
+    "./test_crates/inherent_method_missing/": [
+        {
+            "current_parameter_count": List([]),
+            "method_name": String("will_be_removed_associated_fn"),
+            "method_visibility": String("public"),
+            "name": String("Foo"),
+            "non_matching_span_begin_line": List([]),
+            "non_matching_span_filename": List([]),
+            "old_parameter_count": Uint64(0),
+            "path": List([
+                String("inherent_method_missing"),
+                String("Foo"),
+            ]),
+            "span_begin_line": Uint64(7),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "current_parameter_count": List([]),
+            "method_name": String("will_be_removed_method"),
+            "method_visibility": String("public"),
+            "name": String("Foo"),
+            "non_matching_span_begin_line": List([]),
+            "non_matching_span_filename": List([]),
+            "old_parameter_count": Uint64(1),
+            "path": List([
+                String("inherent_method_missing"),
+                String("Foo"),
+            ]),
+            "span_begin_line": Uint64(9),
+            "span_filename": String("src/lib.rs"),
+            "visibility_limit": String("public"),
+        },
+    ],
```

Comment:
Interesting! These are false-positive on account of being duplicate lints: these methods were removed, but they are being reported as having changed parameter counts as well.

To move this PR along, let's just file an issue for this and resolve it in a future PR.

Reply:
Fixed in #220.